projects/rocprofiler-systems/source/bin/rocprof-sys-avail/common.cpp

```diff
@@ -396,3 +399,46 @@ file_exists(const std::string& _fname)
 }
 
 //--------------------------------------------------------------------------------------//
+
+void
+filter_operations(const std::string& env_var_name, std::vector<std::string>& choices)
+{
+    // Define operations to exclude for each environment variable that supports operations
+    // The exclusions apply to _OPERATIONS, _OPERATIONS_EXCLUDE, and
+    // _OPERATIONS_ANNOTATE_BACKTRACE
+    static const std::unordered_map<std::string, std::unordered_set<std::string>>
+        exclusions = {
+            { "ROCPROFSYS_ROCM_OMPT_OPERATIONS",
+              { "omp_callback_functions", "omp_thread_end" } },
+        };
+
+    std::string      base_name     = env_var_name;
+    constexpr size_t exclude_len   = 8;   // "_EXCLUDE"
+    constexpr size_t backtrace_len = 19;  // "_ANNOTATE_BACKTRACE"
```

Comment:
```suggestion

    constexpr auto exclude_str = "_EXCLUDE";
    constexpr auto annotate_backtrace_str = "_ANNOTATE_BACKTRACE";
    constexpr size_t exclude_len   = sizeof(exclude_str);
    constexpr size_t backtrace_len = sizeof(annotate_backtrace_str );
```

Also done in compile time, but you will have better readability. Also, you will be able to use `exclude_str` and  `annotate_backtrace_str ` variables in code below.

Reply:
`sizeof(exclude_str)` and `sizeof(annotate_backtrace_str)` both produce 8 because it decayed to a pointer and it returns the size of said pointer. 

I'll use:
```
    constexpr size_t exclude_len   = sizeof("_EXCLUDE") - 1;
    constexpr size_t backtrace_len = sizeof("_ANNOTATE_BACKTRACE") - 1;
```